### DISCUSSION TITLE
Are there plans to make the Copilot protocol open-source?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18925] [Copilot Conversations]
I'm not talking about making sharing the dataset or making the extensions open-source, just the protocol so people can port it in to other editors that GH doesn't have plans to port it to.

**Comment by zerolfx** [comment]
I successfully port Copilot from https://github.com/github/copilot.vim to https://github.com/zerolfx/copilot.el. The protocol is similar to language server protocol and has a bit more APIs for authentication.

**Reply by ta32** [reply]
The copilot.vim repo looks official - I wonder why they haven't open-source the vscode and the jet-brains extensions...
